### DISCUSSION TITLE
Preset Update

### DISCUSSION THREAD

**Discussion by rodrigocypriano** [rsjaffe/MIDI2LR#615] [Q&A]
Hi,

Is there a way to update the presets and not have to go to the reveal preset and re-enter the preset?

I have temperature presets that I change with each event, but when I update the preset the message appears:

[string "ClientUtilities.lua"]:350: attempt to
index upvalue 'preset' (a nil value)

Sorry for google english

Mac os Monterrey - Mid2LR 5.2.0.0

**Comment by rsjaffe** [comment]
Are you talking about develop presets? MIDI2LR uses the Lightroom internal ID for the preset. If that changes, MIDI2LR can't find the preset any more.
